src/Tribe/Service_Providers/Context.php

```diff
@@ -274,6 +283,11 @@ static function ( $post_type ) {
 				],
 				'event_post_type'      => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								return ! empty( tribe_get_request_var( TEC::POSTTYPE, false ) );
+							}
```

Comment:
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma
* :no_entry_sign: Comma required after last value in array declaration

<sub>hash: 034aaf95908bdab7fb4d6eba61569f0a</sub>